dada2_inference.R

```diff
@@ -0,0 +1,299 @@
+#!/usr/bin/env Rscript
+
+# Read in package to read in command-line options.
+library("optparse")
+
+version <- "1.0"
+
+option_list <- list(
+
+  make_option(c("-f", "--f_path"), type="character", default=NULL,
+              help="Location of forward reads (required)." , metavar="path"),
+  
+  make_option(c("--seed"), type="integer", default=NULL,
+              help="Random seed to make command reproducible (required).", metavar = "integer"),
```

Comment:
Why is a random seed required? Should this be optional?

Reply:
I think this should be required otherwise pipelines wont be reproducible, but I can set this to be optional!

Reply:
I would prefer it to be optional. Hopefully end results don't vary too drastically based on the initial seed.

Reply:
It might be nice to output the seed used somewhere, in case the user wants it for later. Not a requirement, of course.

Reply:
I changed it to be optional; if the user doesn't set it then the seed isn't explicitly set. If verbose=TRUE and the random seed is set it'll be printed to the logfile.